RELEASING.md

```diff
@@ -37,8 +37,12 @@ These steps are for creating a new major/minor release, with the expectation tha
        version: vX.Y
    ```
 
+4. Update the `RELEASE_STREAM` variable at the top of calico_node/Makefile to the new major/minor version.
+
 4. Test the changes locally then open a pull request, make sure it passes CI, then merge.
 
+5. run `make -C calico_node release` to create container images for calico/node. Push them by following the instructions that the make command prints.
```

Comment:
Can't we just have all of the makefiles defined in the root so we don't need the -C?